src/vxworks/mod.rs

```diff
@@ -25,23 +25,11 @@ pub type c_ulonglong = u64;
 pub type intmax_t = i64;
 pub type uintmax_t = u64;
 
-cfg_if! {
-    if #[cfg(target_pointer_width = "32")] {
-        pub type size_t = c_uint;
-        pub type ssize_t = c_int;
-        pub type ptrdiff_t = c_int;
-        pub type intptr_t = c_int;
-        pub type uintptr_t = c_uint;
-    } else if #[cfg(target_pointer_width = "64")] {
-        pub type size_t = c_ulonglong;
-        pub type ssize_t = c_longlong;
-        pub type ptrdiff_t = c_longlong;
-        pub type intptr_t = c_longlong;
-        pub type uintptr_t = c_ulonglong;
-    } else {
-        // Unknown target_pointer_width
-    }
-}
+pub type size_t = usize;
```

Comment:
How is `size_t` defined in the VxWorks headers ? Can you copy that part of the header ?

Reply:
It is defined like this:
#ifdef _WRS_CONFIG_LP64
typedef unsigned long long size_t;
#else
typedef unsigned int size_t;
#endif

Reply:
How are uintptr_t and intptr_t defined ?

Reply:
#ifdef _WRS_CONFIG_LP64
typedef long long intptr_t;
typedef unsigned long long uintptr_t;
#else
typedef int intptr_t;
typedef unsigned int uintptr_t;
#endif

Reply:
So I suggest defining `uintptr_t = usize` and `intptr_t = isize`, and using `uintptr_t` to define `size_t`, and `intptr_t` to define `ssize_t`.

Reply:
How about ptrdiff_t?

Reply:
It's ok to make it `isize` I think.